pkg/router/template/router.go

```diff
@@ -398,6 +399,12 @@ func (r *templateRouter) AddRoute(id string, route *routeapi.Route, host string)
 	config := ServiceAliasConfig{
 		Host: host,
 		Path: route.Spec.Path,
+		Hash: backendKey,
+	}
+
+	// Possibly hash the name for later use in obsuring cookie ID
+	if os.Getenv("COOKIE_PREFIX_MAIN") + os.Getenv("COOKIE_PREFIX_EDGE") + os.Getenv("COOKIE_PREFIX_REENCRYPT") != "" {
+		config.Hash = fmt.Sprintf("%x", md5.Sum([]byte(backendKey)))
```

Comment:
I think it is, others please chime in, since the thing being hashed is internal data that is generated as part of normal operation and is not a constant in the code. You have to already know the configured project and route in order to find the matching hash.